Formula/gusto-lima.rb

```diff
@@ -12,11 +12,21 @@ class GustoLima < Formula
   depends_on "docker-credential-helper-ecr"
 
   def install
-    bin.install_symlink Formula["colima"].bin/"colima" => "gusto-lima"
+    gusto_colima_script = Tempfile.new(["colima_gusto", ".sh"])
+    gusto_colima_script.write <<~SCRIPT
+      #!/bin/bash
+      colima -p gusto $@
```

Comment:
It's good to make sure to quote `$@`.

I also like to exec to replace the current process with the new command, if all a script is doing is preparing to run something else.

```suggestion
      exec colima -p gusto "$@"
```